### PR TITLE
Config from py

### PR DESCRIPTION
**Pull request by wichersn**
Allows the reward and ammo parameters to be configured from gin instead of from the WAD file. Also simplifies the code to add channels

### CODE REVIEW

**Comment by ricvolpe** [comment]
@wichersn sadly I am not able to succesfully run the code locally because it seems I don't have the rigth versions of the required packages. Are you by any chance able to add to the repository a list of versioned required packages (e.g. `requirements.txt`) so that I can install them and run the code locally?

**Comment by ricvolpe** [comment]
Ok, I managed to run the agent training scripts using
* tensorflow 2.2
* tensorflow probability 0.10
I also managed to finally play the game using pygame
